rpcs3/Emu/RSX/Overlays/overlay_trophy_notification.cpp

```diff
@@ -71,11 +74,11 @@ namespace rsx
 				return;
 			}
 
-			if (((t - creation_time) / 1000) > 7500)
+			if (((t - creation_time) / 1000) > 5000)
 			{
 				if (!sliding_animation.active)
 				{
-					sliding_animation.end = { -f32(frame.w), 0, 0 };
+					sliding_animation.end = { -f32(frame.w*1.25), 0, 0 };
```

Comment:
I suppose you want to speed up the leaving animation?

Anyway. it's missing some spaces
```suggestion
					sliding_animation.end = { -f32(frame.w * 1.25f), 0, 0 };
```

Reply:
This fixes the notification not fully sliding back.

Reply:
I think the animation is broken because you moved the notification position to the right.
It's probably better to use that instead of some arbitrary factor

Reply:
I sent a commit without this part. Also fixed the formatting. It's ready to merge.